lib/opensusebasetest.pm

```diff
@@ -979,6 +979,9 @@ sub wait_boot {
     # shutting down or booting up
     $self->{in_wait_boot} = 1;
 
+    # wait grub page for powerVM
+    $self->wait_grub(bootloader_time => $bootloader_time) if (is_pvm);
```

Comment:
Not sure it is a good idea to introduce this pvm specific thing here. Could you please explain in the comment in detail why pvm should need this? Also please use just `if is_pvm` without the parentheses

Reply:
Between select_console powerhmc-ssh and        $self->handle_grub(bootloader_time => $bootloader_time, in_grub => $in_grub);  We need wait grub page show up. Before we don't have wait_grub, it failed to catch Please check https://10.160.0.207/tests/4330426. Then I debug to add sleep after 'select_console powerhmc-ssh', you can check https://openqa.nue.suse.com/tests/4339637#step/yast2_migration/18. After we get grub page then run  $self->handle_grub(bootloader_time => $bootloader_time, in_grub => $in_grub);  it can work.

Reply:
hm, I wonder, is there no other code that calls `wait_boot` for pvm? In any case, thanks for explaining but please explain in the *in-file* comment why pvm should need this (please not only how you fixed it, but why it is needed for pvm but not for others).

Reply:
OK, I will add comment for those code